src/main/java/org/vietabroader/controller/ConnectController.java

```diff
@@ -0,0 +1,50 @@
+package org.vietabroader.controller;
+
+import org.vietabroader.GoogleAPIUtils;
+import org.vietabroader.model.GlobalState;
+import org.vietabroader.model.VASpreadsheet;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ConnectController implements Controller {
+
+    private JButton btnConnect;
+    private JTextField txtConnect;
+
+    public ConnectController setConnect(JButton btn, JTextField txt) {
```

Comment:
split this into 2 methods: `setButtonConnect` and `setTextSpreadsheetID`
We `return this` for each method so that we can cascade them like this:
```
connectController.setButtonConnect(btn)
                              .setTextSpreadsheetID(text);
```

For only 2 components, this might look redundant. But imagine 5 or even more components, if you do:
```
connectController.setComponents(btnX, btnY, textA, textB, texC);
```
How do you know what btnX is and how do you know if it btnX should be in the first argument or the second one? When you cascade it, it looks much nicer:
```
connectController.setButtonX(btnX)
                              .setTextA(textA)
                              .setButtonY(btnY)
                              .setTextB(textB)
                              .setTextC(textC);
```
Note that the order now does not matter anymore and the code is more descriptive.

We want to keep this style consistent throughout all controllers so do this even if there are only 2 components.